modules/data-entry/src/main/frontend/src/dataHomepage/UserDashboard.jsx

```diff
@@ -16,163 +16,172 @@
-        <Grid item lg={12} xl={6}>
-          <Card>
-            <CardHeader
-              title={
-                  <Button className={classes.cardHeaderButton}>
-                    Incomplete Forms
-                  </Button>
-              }
-            />
-            <CardContent>
-              <LiveTable
-                columns={columns}
-                customUrl='/Forms.paginate?fieldname=statusFlags&fieldvalue=INCOMPLETE'
-                defaultLimit={10}
-                joinChildren="lfs:Answer"
-                filters
-                entryType={"Form"}
-                actions={actions}
-              />
-            </CardContent>
-          </Card>
-        </Grid>
-        {questionnaires.map( (questionnaire) => {
-          const customUrl='/Forms.paginate?fieldname=questionnaire&fieldvalue='
-            + encodeURIComponent(questionnaire["jcr:uuid"]);
-          return(
-            <Grid item lg={12} xl={6} key={questionnaire["jcr:uuid"]}>
-              <Card>
-                <CardHeader
-                  title={
-                    <Link href={`/content.html/Forms?questionnaire=${questionnaire["jcr:uuid"]}`}>
-                      <Button className={classes.cardHeaderButton}>
-                        {questionnaire["title"]}
-                      </Button>
-                    </Link>
-                  }
-                  action={
-                    <NewFormDialog presetPath={questionnaire["@path"]}>
-                      New form
-                    </NewFormDialog>
-                  }
-                  classes={{
-                    action: classes.newFormButtonHeader
-                  }}
-                />
-                <CardContent>
-                  <LiveTable
-                    columns={columns}
-                    customUrl={customUrl}
-                    defaultLimit={10}
-                    joinChildren="lfs:Answer"
-                    questionnaire={questionnaire["@path"]}
-                    entryType={questionnaire["title"]}
-                    filters
-                    actions={actions}
-                    />
-                </CardContent>
-              </Card>
+        {
+          dashboardExtensions.map((extension, index) => {
+            let Extension = extension["lfs:extensionRender"];
+            return <Grid item lg={12} xl={6} key={"extension-" + index}>
+              <Extension />
             </Grid>
-          )
-        })}
+          })
+        }
       </Grid>
+      <Dialog open={open} onClose={onClose}>
+        <DialogTitle className={classes.dialogTitle}>New</DialogTitle>
+        <DialogContent dividers>
+          <MaterialTable
+            columns={[
+              { field: 'lfs:extensionName' },
+            ]}
+            data={creationExtensions}
+            options={{
+              search: true,
+              pageSize: rowCount,
+              showTitle: false,
+              paging: creationExtensions.length > 5,
```

Comment:
Shouldn't it be
```suggestion
              paging: creationExtensions.length > rowCount,
```
?

Reply:
I thought about doing that, but was worried about the user losing the ability to lower the rowCount if they set it too high.
Eg. 19 entries, set rowCount to 20 and then having pagination and the option to change rowcount hidden

Reply:
I see, that's fine then.